stack/base/before-notebook.d/40_prepare-aiida.sh

```diff
@@ -74,6 +74,8 @@ load_computer('${computer_name}').set_minimum_job_poll_interval(${job_poll_inter
 else
 
   # Migration will run for the default profile.
+  ## clean up stale PID-files -> .aiida/daemon/circus-{profile-name}.pid
+  rm -f "/home/${NB_USER}/.aiida/daemon/circus-${AIIDA_PROFILE_NAME}.pid"
```

Comment:
@danielhollas we suppose the user will just use the ${AIIDA_PROFILE_NAME} profile, right? 
I am not sure if we should also consider the possibility to define other profiles, so we should consider also to clean their pid(s)

Reply:
Let's just do:

```suggestion
  rm -f "/home/${NB_USER}/.aiida/daemon/circus-*.pid"
```